controllers/ControllerComment.php

```diff
@@ -47,30 +41,32 @@ public function addCommentPost(){
 			$good = $this->_commentManager->addComment($userId, $comment, $postId['id']);
 
 			if ($good){
-				header("Location: http://".$this->_server."/Post/".$postId['id']);
+				header("Location: http://".$this->_server."/Post/viewPost/".$postId['id']);
```

Comment:
Codacy found an issue: [The use of function header() is discouraged](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=6061935)